packages/date-helpers/__tests__/date-helpers.test.ts

```diff
@@ -35,12 +91,22 @@ describe('DateHelpers', () => {
     ).toBe('10/21/2019 00:00:00');
   });
 
-  it('convertDateToMonthDayYearString should format dates correctly', () => {
+  it('formatDateLikeToUsLocaleMediumDateString should format dates correctly', () => {
     expect(DateHelpers.formatDateLikeToUsLocaleMediumDateString(output)).toBe(
       'Oct 21, 2019',
     );
     expect(
       DateHelpers.formatDateLikeToUsLocaleMediumDateString('2019-10-21'),
     ).toBe('Oct 21, 2019');
   });
+
+  it('formatDateLikeToString should return an empty string if the input is an empty string', () => {
```

Comment:
It's strange that it doesn't throw in this case though.

Reply:
If we no longer accept empty strings here, it should be a breaking change. It should be handled in a separate PR.